phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -90,6 +91,113 @@ public void testStartKeyStopKey() throws SQLException {
         assertTrue(splits.size() > 0);
     }
 
+    @Test
+    public void testCreateAlterTableWithDuplicateColumn() throws Exception {
+        Properties props = new Properties();
+        int failureCount = 0;
+        int expectedExecCount = 0;
+        String tableName = generateUniqueName();
+        String viewName = generateUniqueName();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            try {
+                conn.createStatement().execute(String.format("CREATE TABLE %s"
```

Comment:
nit: Would be good if before each case there was a brief comment explaining it -- e.g "Same column name, different column type" for the third one.

Reply:
Definitely, on it.

Reply:
Done. Thanks @gjacoby126